launcher/modplatform/flame/FlameModIndex.cpp

```diff
@@ -51,12 +41,20 @@ void FlameMod::loadIndexedPackVersions(ModPlatform::IndexedPack& pack,
     for (auto versionIter : arr) {
         auto obj = versionIter.toObject();
 
-        auto versionArray = Json::requireArray(obj, "gameVersion");
+        auto versionArray = Json::requireArray(obj, "gameVersions");
         if (versionArray.isEmpty()) { continue; }
 
         ModPlatform::IndexedVersion file;
+
+        bool is_valid_fabric_version = false;
+
         for (auto mcVer : versionArray) {
-            file.mcVersion.append(mcVer.toString());
+            auto str = mcVer.toString();
+
+            if (str.indexOf('.') > -1)
+              file.mcVersion.append(mcVer.toString());
+            
+            if (str.toLower() == "fabric") is_valid_fabric_version = true;
```

Comment:
do we actually need this? we already request with modLoaderType

Reply:
ahaha that is true we don't need this